### ISSUE TITLE
Only show release channels in the system settings if there's more than one

### ISSUE THREAD

**Issue by tsimonq2** [ubports/ubuntu-touch#497]
- Device: hammerhead
- Channel: xenial/devel
- Build: 273

### Steps to reproduce
On 16.04/devel, go to System Settings -> Updates -> Update settings -> Channels.

### Expected behavior

A single entry should be shown for the one Xenial channel that is active.

Otherwise, the whole Update settings option should just be absent, as there's nothing to see there.

### Actual behavior

Tapping on it does nothing.

**Comment by UniversalSuperBox** [comment]
Hello Mr. Tee-Simon-Queue-two,

There aren't any channels in Xenial yet. However, when there are channels available (such as when we near release of 16.04 images), this section should populate.

**Comment by tsimonq2** [comment]
Right, but why is that option even there anyways if there's no channels?

**Comment by UniversalSuperBox** [comment]
Because Xenial is as of now unreleased to anyone but developers, and when it is released there will be channels. There's no reason to remove something when we plan on adding it again in a month.

**Comment by tsimonq2** [comment]
What about when we move to something like Bionic and then from there to the f-cycle (20.04)... this won't be the only time this will be the case.

**Comment by mariogrip** [comment]
True, at this moment it's not really a problem. Once we create rc and stable this will be fixed,

**Comment by mariogrip** [comment]
Ops, wrong button :P

**Comment by mariogrip** [comment]
But it's true that it should say something like "there is only devel" or something like that. Since this will indeed be a problem later for us

**Comment by NeoTheThird** [comment]
This is not a xenial-issue, but a general one. If there is only one channel available, the option should be hidden.

**Comment by nfsprodriver** [comment]
https://github.com/ubports/system-settings/pull/42

**Comment by NeoTheThird** [comment]
Wow, that was fast! PR looks fine, just requesting review from @mariogrip if we really want this, or rather just allow users to move freely among all the channels even of different ubuntu bases.

**Comment by nfsprodriver** [comment]
That would also be a nice option ;)

**Comment by NeoTheThird** [comment]
Does not seem to work.

**Comment by Flohack74** [comment]
Hm not sure what you mean with does not work, it only shows Development there, I think this is ok to assure people they are on this or that channel. Tapping on it leads to a single item in a list, and I think its cosmetic. This is eventually shiftable to OTA-5
